new-components/enrichers/reachability/cdxgen-atom-runner/main.go

```diff
@@ -0,0 +1,161 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"log"
+	"log/slog"
+	"os"
+	"os/exec"
+	"path"
+	"strings"
+
+	"github.com/go-errors/errors"
+	langs "github.com/smithy-security/pkg/detect-repo-languages"
+)
+
+var (
+	sourceLocation  string
+	scratchLocation string
+)
+
+type SupportedLanguage int
+
+const (
+	C SupportedLanguage = iota
+	Java
+	Javascript
+	Typescript
+	PHP
+	Python
+	Ruby
+)
+
+var (
+	langName = map[SupportedLanguage]string{
+		Java:       "java",
+		Javascript: "javascript",
+		Typescript: "javascript",
+		PHP:        "php",
+		Python:     "python",
+		Ruby:       "ruby",
+	}
+	toLang = map[string]SupportedLanguage{
+		"java":                 Java,
+		"java properties":      Java,
+		"java server pages":    Java,
+		"java template engine": Java,
+		"javascript":           Javascript,
+		"javascript+erb":       Javascript,
+		"typescript":           Typescript,
+		"php":                  PHP,
+		"python":               Python,
+		"python console":       Python,
+		"python traceback":     Python,
+		"ruby":                 Ruby,
+	}
+)
+
+func main() {
```

Comment:
Couldn't we add this logic to the component itself to make sure that it's served by the SDK?